include/FanFicArchive.py

```diff
@@ -0,0 +1,85 @@
+from bs4 import BeautifulSoup; from urllib import request; from re import match; import requests
```

Comment:
Usually you would format the code according to the prevailing style in the rest of the repository; failing that, PEP 8.

Reply:
Well said